gammapy/datasets/core.py

```diff
@@ -505,7 +505,8 @@ def info_table(self, cumulative=False):
         Parameters
         ----------
         cumulative : bool
-            Cumulate info across all observations. Default is False.
+            Cumulate info across all observations. If True, all the information that depend on a
+            model, such as 'stat_sum', will be lost. Default is False.
```

Comment:
Now that I think about it, this docstring is not good. Because in the case of OnOff datasets, the stat_sum will be the stat_sum of the stacked dataset (since it's just WStat of the dataset). But in the case where it's not an OnOff datasets (MapDataset, SpectrumDataset), then you effectively loose the stat_sum because the models are not stacked.